pkg/api/types.go

```diff
@@ -1362,6 +1362,8 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+	// The SeccompProfile is the seccomp profile this pod should run under.
```

Comment:
This should be a name right now to match the proposal, I do not want to specify these profiles inline -- it makes the policy much harder to write.